src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3217,6 +3217,158 @@ paths:
+        403:
+          description: Unauthorized - missing permissions to request snapshot.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        404:
+          description: The group members cannot be found or you do not have access
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        500:
+          description: An unexpected error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+    post:
+      tags:
+        - SnapshotAccessRequest
+      description: Adds a member to the DAC group
+      operationId: addSnapshotAccessRequestGroupMember
+      parameters:
+        - $ref: '#/components/parameters/Id'
+        - name: memberEmail
+          in: query
+          description: The email of the user to add
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PolicyMemberRequest'
+        required: true
+      responses:
+        200:
+          description: The member has been added to the group
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SnapshotAccessRequestMembersResponse'
+        400:
+          description: Bad request - invalid id, snapshotRequest, or badly formed IamResourceType.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        401:
+          description: Unauthorized - dataset does not exist or missing permissions to view.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        403:
+          description: Unauthorized - missing permissions to request snapshot.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        404:
+          description: No dataset found that meets the request parameters.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+        500:
+          description: An unexpected error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/snapshotAccessRequests/{id}/members/{memberEmail}:
+    delete:
+      tags:
+        - SnapshotAccessRequest
+      description: Removes a member from the DAC group
+      operationId: deleteSnapshotAccessRequestGroupMember
+      parameters:
+        - $ref: '#/components/parameters/Id'
+        - name: memberEmail
```

Comment:
Since we don't have an openapi type for email (e.g. we could define a type with a regex to match only emails) I think we need to validate the request. One way to do that is to use `ValidationUtils.isValidEmail()`.